### PR TITLE
Release 0.0.3

### PR DESCRIPTION
**Pull request by frondeus**

### CODE REVIEW

**Review by luke-biel — DISMISSED** [review]

File: .github/workflows/testing.yml [hunk 1]
```diff
@@ -41,55 +41,3 @@ jobs:
         with:
           command: test
           args: --workspace
-
```
Comment (USSAMOH2) [inline .github/workflows/testing.yml:44]: Why get rid of the integration testing?

File: examples/deploy/docker-compose.yml [hunk 2]
```diff
@@ -6,109 +6,150 @@
...
-  schema_registry1:
-    image: schema_registry:latest
+  schema_registry:
+    image: cdl-schema-registry:latest
     build:
       context: ../..
       dockerfile: Dockerfile
       args:
         - BIN=schema-registry
+        - ENV=DEV
```
Comment (USSAMOH2) [inline examples/deploy/docker-compose.yml:56]: Why use debug builds?

File: query-service/src/psql.rs [hunk 3]
```diff
@@ -74,8 +77,8 @@ impl PsqlQuery {
     fn collect_id_payload_rows(rows: Vec<Row>) -> HashMap<String, Vec<u8>> {
         rows.into_iter()
             .map(|row| {
-                let object_id = row.get::<usize, String>(0);
-                let value = row.get::<usize, String>(1).into_bytes();
+                let object_id = row.get::<usize, Uuid>(0).to_string();
```
Comment (USSAMOH2) [inline query-service/src/psql.rs:80]: This seems like it is just slower than simply retrieving as a string. We don't get more validation here, as non-UUID values cause a panic here.

File: query-service/src/psql.rs [hunk 4]
```diff
@@ -74,8 +77,8 @@ impl PsqlQuery {
...
+                let value = row.get::<usize, Value>(1).to_string().into_bytes();
```
Comment (USSAMOH2) [inline query-service/src/psql.rs:81]: Instead of to_string and into_bytes, you should use serde_json::to_vec

**Review by USSAMOH2 — DISMISSED** [review]

**Review by luke-biel — APPROVED** [review]

**Comment by luke-biel** [comment]
Shall we remove building on develop pushes?
@frondeus

**Pull request merged**